gif-screencast.el

```diff
@@ -64,6 +67,16 @@ various programs run here."
   :group 'gif-screencast
   :type 'string)
 
+(defvar gif-screencast-cropping-program "mogrify"
+  "A program for cropping the screenshots.")
+
+(defcustom gif-screencast-cropping-args '("-verbose")
```

Comment:
Why verbose?

Reply:
Actually, it means nothing. I just used the value as an example.
Replaced `'("-verbose")` with `nil` but also update `:type '(repeat string))` so that user can understand how to use this variable.